CHANGES.rst

```diff
@@ -7,6 +7,7 @@
 ==================
 
 - Add support for python 3.7 and 3.8
+- Greenlet dependency is only required on CPython. See :issue:`2`
```

Comment:
This file is rendered on PyPI, so Sphinx extensions aren't valid. Try running `restview --long —pypi`.

Reply:
Thanks for sharing `restview`